subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/DependencyManagementBuildScopeServices.java

```diff
@@ -200,6 +209,13 @@ ExternalResourceFileStore createExternalResourceFileStore(CacheScopeMapping cach
         return new ExternalResourceFileStore(cacheScopeMapping.getBaseDirectory(null, "external-resources", VersionStrategy.SharedCache), new TmpDirTemporaryFileProvider());
     }
 
+    TextResourceLoader createTextResourceLoader(ExternalResourceFileStore resourceFileStore, RepositoryTransportFactory repositoryTransportFactory) {
+        HashSet<String> schemas = Sets.newHashSet("https", "http");
+        RepositoryTransport transport = repositoryTransportFactory.createTransport(schemas, "http auth", Collections.<Authentication>emptyList());
+        ExternalResourceAccessor externalResourceAccessor = new DefaultExternalResourceAccessor(resourceFileStore, transport.getResourceAccessor());
+        return new DefaultUriTextResourceLoader(externalResourceAccessor, schemas);
```

Comment:
Oh, I just realized that this means we'll see the downloads come through build scans! 🎉